app/models/school_local_authority_district.rb

```diff
@@ -1,5 +1,27 @@
 # frozen_string_literal: true
 
+# == Schema Information
+#
+# Table name: school_local_authority_districts
+#
+#  id                          :uuid             not null, primary key
+#  end_year                    :integer
+#  start_year                  :integer          not null
+#  created_at                  :datetime         not null
+#  updated_at                  :datetime         not null
```

Comment:
I think `id`, `created_at` and `updated_at` are gonna be on every single model we have. Not sure it is worth putting them in here - they might obscure the meaningful things.